src/test/resources/data/StudentFeedbackQuestionSubmitPageUiTest.json

```diff
@@ -29,6 +29,7 @@
             "name":"Teammates Test",
             "email":"SFQSubmitUiT.instr@gmail.tmt",
             "role":"Co-owner",
+            "isDisplayedToStudents":true,
```

Comment:
@unyoungwax, When I manually test studentFeedbackQuestionSubmissionEditPage on master, the hidden instructors are showing up. So this modification for the pr is expected then, since you fixed this too